### ISSUE TITLE
how to extract facial regions?

### ISSUE THREAD

**Issue by wolo-wolo** [zimyang/MaskRelation#3]
Thank you for your great work. Fig. 1. in the paper shows facial regions segmentation map and treats them as nodes.  How is the seg map obtained, and is it implicit in the code to obtain the facial region?

**Comment by zimyang** [comment]
Facial regions can be extracted by face detectors such as Dlib, MTCNN, etc. Partial regions are masked by the proposed Masked Relation Learner. The implementation is _class Model_ on Line 33 in [network.py](https://github.com/zimyang/MaskRelation/tree/main/Pytorch/training/network.py).

**Comment by anbreenkausar** [comment]
Thankyou @zimyang for your great work. I am working on it. I used Dlib to crop faces from videos but frame quality gets damaged and image gets blurred! Please guide me what should I do to continue work on your paper. Thanks in advance.

**Comment by dt140120** [comment]
can u share the training dataset ?
